docs/artifacts.asciidoc

```diff
@@ -0,0 +1,8 @@
+[[ecs-artifacts]]
+=== ECS Experimental Artifacts
```

Comment:
Why call the artifacts experimental?

The tooling I consider experimental, but the reference ECS artifacts themselves.

Reply:
A fair point. I was thinking too narrowly about the Elasticsearch templates being intended as a starting point for experimental. I'll adjust the wording a bit with this in mind.